tldextract/tldextract.py

```diff
@@ -208,15 +211,20 @@ def __call__(self, url):
         registered_domain = ".".join(labels[:suffix_index])
         tld = ".".join(labels[suffix_index:])
 
-        if not tld and netloc and netloc[0].isdigit():
+        if not tld and netloc:
             try:
-                is_ip = socket.inet_aton(netloc)
+                is_ip = socket.inet_pton(socket.AF_INET, netloc)
                 return ExtractResult('', netloc, '')
             except (AttributeError, UnicodeError):
                 if IP_RE.match(netloc):
                     return ExtractResult('', netloc, '')
             except socket.error:
                 pass
+            try:
+                is_ip = socket.inet_pton(socket.AF_INET6, netloc)
```

Comment:
In your description, you're saying this `try` is unnecessary? Line 230 will do the right thing?